akka-sse/src/main/scala/de/heikoseeberger/akkasse/ServerSentEvent.scala

```diff
@@ -21,6 +21,12 @@ import scala.annotation.tailrec
 
 object ServerSentEvent {
 
+  sealed trait EventIdValue
+  case object ResetId extends EventIdValue
+  final case class Id(id: String) extends EventIdValue {
```

Comment:
It does, even though [a bit hidden](http://www.w3.org/TR/eventsource/#event-stream-interpretation) 

> The third block fires an event with data "second event", and also has an "id" field, this time with no value, which resets the last event ID to the empty string (meaning no Last-Event-ID header will now be sent in the event of a reconnection being attempted).

Just saw that my implementation is wrong because it sends a colon instead of a plain `id\n` line.